modules/fontrendering/include/modules/fontrendering/textrenderer.h

```diff
@@ -316,7 +316,7 @@ class IVW_MODULE_FONTRENDERING_API TextRenderer {
     int fontSize_;        //<! font size in pixel
     double lineSpacing_;  //!< spacing between two lines in percent (default = 0.2)
 
-    const int glyphMargin_;
+    static constexpr int glyphMargin_ = 2;
```

Comment:
it was a `const 2` previously set in the constructor. And the name is exactly what it is. The margin/spacing around a glyph in the atlas to prevent interpolation artifacts.

Reply:
fixed.